tiledb/attribute.py

```diff
@@ -46,15 +46,13 @@ def __init__(
         :raises: :py:exc:`tiledb.TileDBError`
 
         """
-        self._ctx = ctx or default_ctx()
-        _dtype = None
-
         if _capsule is not None:
-            return super().__init__(self._ctx, _capsule)
+            return super().__init__(ctx, _capsule)
```

Comment:
Shouldn't these still be `self._ctx`?

Reply:
There is no `self._ctx` yet, it is set in `super().__init__`